canmatrix/dbc.py

```diff
@@ -455,9 +455,9 @@ class FollowUps(object):
                                      dbcImportEncoding),
                                  receiver=receiver,
                                  multiplex=multiplex)
-                if not tempSig.is_little_endian:
+                #if not tempSig.is_little_endian:
```

Comment:
Yeah will do:

The higher level code uses the default startbit location (most signifigant bit) to calculate big endian values. This extra logic is not necessary and causes problems.